src/Microsoft.AspNet.Server.Kestrel/project.json

```diff
@@ -1,9 +1,11 @@
 {
-    "version": "1.0.0-*",
-    "description": "ASP.NET 5 cross platform development web server.",
-    "dependencies": {
-        "Microsoft.Framework.Runtime.Interfaces": "1.0.0-*"
-    },
+	"version": "1.0.0-*",
```

Comment:
Thanks @Eilon , my editor settings causes this problem so I just set it to keep spaces.